lightning/src/ln/channel.rs

```diff
@@ -2278,19 +2277,18 @@ impl<SP: Deref> PendingV2Channel<SP> where SP::Target: SignerProvider {
 			&funding_inputs_prev_outputs, &funding_outputs,
 			self.dual_funding_context.funding_feerate_sat_per_1000_weight,
 			self.context.holder_dust_limit_satoshis,
-		).map_err(|err| APIError::APIMisuseError {
-			err: format!("Insufficient inputs, cannot cover intended contribution of {} and fees; {}",
-				self.dual_funding_context.our_funding_satoshis, err
-			),
-		})?;
+		).map_err(|err| ChannelError::Warn(format!(
+			"Insufficient inputs, cannot cover intended contribution of {} and fees; {}",
+			self.dual_funding_context.our_funding_satoshis, err
+		)))?;
```

Comment:
Related to my other comment, this message is intended more for the user not our counterparty, IIUC. Likewise for other messages that would be sent to the counterparty as a `warning`. I guess what is sent or not to the counterparty would depend on the calling context.

Reply:
See above, https://github.com/lightningdevkit/rust-lightning/pull/3443#discussion_r2000575422